src/components/TodosFilter/TodosFilter.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useContext } from 'react';
+import cn from 'classnames';
+import { TodosContext } from '../../store/store';
+import { Status } from '../../types/enums/Status';
+import { Dispatchers } from '../../types/enums/Dispatchers';
+
+import './TodosFilter.scss';
+
+interface Props {
+  handleSetFilterParam: (param: Status) => void;
+  filterParam: Status;
+}
+
+const shouldRenderButton = (complited: number, total: number): boolean => {
+  if (complited < total) {
+    return false;
+  }
+
+  return true;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  handleSetFilterParam,
+  filterParam,
+}) => {
+  const { state } = useContext(TodosContext);
+  const { dispatch } = useContext(TodosContext);
+
+  const count = state.filter(todo => !todo.completed).length;
+
+  const shouldClearButtonRender = shouldRenderButton(count, state.length);
+
+  return (
+    <footer className="footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${count} items left`}
+      </span>
```

Comment:
![image](https://github.com/mate-academy/react_todo-app/assets/116377279/84ee97b2-05de-4fb5-905c-d2a2b4b193f9)

should be:

![image](https://github.com/mate-academy/react_todo-app/assets/116377279/9e501c1c-908f-48c3-8fcf-953ab4f50fa3)

Reply:
Thanks, I didn't notice this.